hedera/settings.py

```diff
@@ -8,7 +8,10 @@
 from .aws import get_ecs_task_ips
 
 
-IS_LTI = (os.environ.get("IS_LTI") == "1")
+try:
+    IS_LTI = bool(int(os.environ.get("IS_LTI", "0")))
```

Comment:
Good catch on the cast to `int` and corresponding try/catch. Otherwise you'd have to use an empty string to get `bool` to evaluate to false.